### PR TITLE
feat/comment-get-all-pagination

### PR DESCRIPTION
**Pull request by fahmirzldi**
Creating get all pagination using paginate method

### CODE REVIEW

File: app/main/model/comment.py [hunk 1]
```diff
@@ -74,9 +75,13 @@ def create_comment(self, data):
...
             raise e
 
-    def get_all_comments(self):
+    def get_all_comments(self, page, count):
         try:
-            comments = self.query.filter_by(visible=True).all()
+            comments = (
+                self.query.filter_by(visible=True)
+                .order_by(Comment.created_at.desc())
+                .paginate(page=page, per_page=count, max_per_page=20, error_out=False)
```
Comment (fatjan) [inline app/main/model/comment.py:83]: what happen if a user request page 1 with count 50?
will it return page 1 only with 20 items?

Comment (fatjan) [reply app/main/model/comment.py:83]: should we set it to `max_per_page=100` instead?

Comment (fahmirzldi) [reply app/main/model/comment.py:83]: Yes it will only return 20

We can set max page to 100 but will set some default per page to 20

Comment (fatjan) [reply app/main/model/comment.py:83]: ok, does it mean you want to update your PR?

Comment (fahmirzldi) [reply app/main/model/comment.py:83]: Yes done, please kindly review

Comment (fahmirzldi) [reply app/main/model/comment.py]: Oh yeah forgot to push, please kindly check again @fatjan

**Comment by fahmirzldi** [comment]
Kindly review too @AchmadQiz @Achmadqizwini

File: tests/controller/test_comment_controller.py [hunk 2]
```diff
@@ -3,7 +3,12 @@
 from unittest.mock import patch
 from app import create_app
 
-comment_data = {"content": "This is a test comment.", "review_id": "1234ABCD"}
+comment_data_single = {"content": "This is a test comment.", "review_id": "1234ABCD"}
```
Comment (AchmadQiz) [inline tests/controller/test_comment_controller.py:18]: i think there's no need to create two variables. If you need just one piece of data, you can access it by accessing index 0 of that variable or create a new different data in that variable. i think that is an example for implementing the DRY (Don't Repeat Yourself) principle

Comment (fahmirzldi) [reply tests/controller/test_comment_controller.py]: Thanks for the suggestion! kindly check for review

**Review by fatjan — APPROVED** [review]

**Pull request merged**